libs/perun/dialogs/src/lib/password-reset-dialog/password-reset-dialog.component.html

```diff
@@ -0,0 +1,45 @@
+<ng-template #spinner>
+  <perun-web-apps-loading-dialog></perun-web-apps-loading-dialog>
+</ng-template>
+<div class="user-theme position-relative">
+  <h1 mat-dialog-title>
+    {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.TITLE' | translate}}
+  </h1>
+  <div class="dialog-container" mat-dialog-content>
+    <p *ngIf="data.login" class="subtitle">
+      <em
+        >{{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.LOGIN' | translate}}:
+        {{data.login}}</em
+      >
+    </p>
+    <p *ngIf="data.namespace" class="subtitle">
+      <em
+        >{{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.NAMESPACE' | translate}}:
+        {{data.namespace}}</em
+      >
+    </p>
+    <perun-web-apps-password-form
+      class="ml-3 me-3"
+      *ngIf="!success && !loading && !error"
+      [formGroup]="newPasswdForm"
+      [namespace]="data.namespace"
+      [language]="language">
+    </perun-web-apps-password-form>
+
+    <div mat-dialog-actions class="ms-auto justify-content-end">
+      <button (click)="close()" mat-stroked-button>
+        {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.CANCEL' | translate}}
+      </button>
+      <button
+        (click)="onSubmit()"
+        [disabled]="newPasswdForm.invalid || newPasswdForm.pending || loading"
+        class="ms-2"
+        color="accent"
+        mat-flat-button>
+        {{'SHARED_LIB.PERUN.COMPONENTS.RESET_PASSWORD_DIALOG.CHANGE' | translate}}
+      </button>
+    </div>
+  </div>
+
+  <mat-spinner *ngIf="loading" class="ms-auto me-auto"></mat-spinner>
```

Comment:
We no longer use loading indication this way. Rather use a "non blocking" variant. You have the template set up at the start of the file it just needs to be used properly. Check out some other dialog (e.g. create vo) to see how it's done.